app/views/users/index.html.erb

```diff
@@ -1,10 +1,19 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  </head>
+<div class="users">
 
-  <body>
-    <h1>Here is a list of users.</h1>
-  </body>
-</html>
+  <ul class="user_list">
+    <% @users.each do |user| %>
+
+      <li class="user_item">
+        <div class="user_img">
+          <img src=<%= user.photo %> alt="user_img">
+        </div>
+        <div class="user_desc">
+          <h2 class="user_name"><%= user.name %></h2>
+          <p class="user_post_number">Number of posts: <%= user.posts_count %></p>
+        </div>
+      </li>
+      
+    <% end %>
+  </ul>
```

Comment:
- [ ] Please match your layout to the given wireframe as per the requirement:
- [ ] Clicking on each user must take me to that user's show page. Please fix this.

yours|given
---|---
![image](https://github.com/binyamolango/Blog-App/assets/91486791/07c18a96-d122-4bdb-b416-660266c888e5)|![blog_root_users](https://github.com/binyamolango/Blog-App/assets/91486791/809c34f8-4050-4ce0-8142-d7e42d1e81cc)